### PR TITLE
[Poetry] Add spec for Poetry

### PR DESCRIPTION
**Pull request by CastaChick**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
feature

**What is the current behavior? (You can also link to an open issue here)**
no spec for poetry

**What is the new behavior (if this is a feature change)?**
add new spec for poetry

**Additional info:**
poetry is package manager for Python

This PR is the same of #200 because of an issue of Github.

### CODE REVIEW

**Review by QuiiBz — APPROVED** [review]
Thanks for this contribution! (Weird GitHub bug for the last PR, it's resolved now)

File: dev/poetry.ts [hunk 1]
```diff
@@ -0,0 +1,603 @@
...
+        },
+        {
+          name: "value",
+          description: "Setting value",
+        },
+      ],
+    },
+    {
+      name: "debug",
+      description: "Debug various elements of Peotry",
```
Comment (cstrnt) [inline dev/poetry.ts:194]: typo

File: dev/poetry.ts [hunk 2]
```diff
@@ -0,0 +1,603 @@
...
+            suggestions: ["requirements.txt"],
+          },
+        },
+        {
+          name: ["-o", "--output"],
+          description: "The name of the output file",
+          args: {
+            name: "filename",
+            description: "Output file name",
+            suggestions: ["requirements.txt"],
```
Comment (cstrnt) [inline dev/poetry.ts:263]: I think rather than having this suggestion, I would use `template: filepaths` to suggest files

**Review by cstrnt — CHANGES_REQUESTED** [review]
Thank you very much for your contribution

**Comment by CastaChick** [comment]
Thank you for your review! I fixed them☺️

**Comment by QuiiBz** [comment]
> Today, when I run `npm run build`, the output file is differ from the previous one. Is it OK?

Looks like it only affects spacing, it looks OK to me.

**Review by cstrnt — APPROVED** [review]
Well done :)

**Pull request merged**